src/bindings/nodejs/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+cmake_minimum_required(VERSION 3.15)
+
+include(FetchContent)
+
+# Fetch node-addon-api from GitHub
+FetchContent_Declare(
```

Comment:
This guarantees ABI stability, so we don't need to care about different NodeJS versions etc - should just work on any relatively new version of Node...

Reply:
Tbh I don't remember exactly - I will verify before making it ready for review, but IIRC it should be compatible even with something super ancient like Node 8.x (current LTS is Node 22)